File: LeetCode Problems/1 to 100/27. Remove Element.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        for (int i = 0; i < nums.size(); i++) {
            if (nums[i] == val) {
                nums.erase(remove(nums.begin(), nums.end(), val), nums.end());
            }
        }
        return nums.size();
    }
};

File: LeetCode Problems/1 to 100/27. Remove Element/27. Remove Element.java
class Solution {
    public int removeElement(int[] nums, int val) {
        List<Integer> list = new ArrayList<>();
        for (int ele : nums) {
            if (ele != val) list.add(ele);
        }
        int n = list.size();
        for (int i = 0; i < n; i++) nums[i] = list.get(i);
        return n;
    }
}
